### DISCUSSION TITLE
Alerts Continuously Firing

### DISCUSSION THREAD

**Discussion by noctividus** [Navid200/xDrip#89] [Alerts]
Does anyone know what "Re-raise every x minutes if unacknowledged" is supposed to do?

**Comment by Navid200** [comment]
If you don't snooze an alert, it will play again after this amount of time.

**Comment by noctividus** [comment]
So, if my alarm beeps continuously every second for say 20 minutes, and my
x=5, does that mean that there is a bug?

**Reply by Navid200** [reply]
It depends on what sound file you have chosen for the alarm.  In other words, if you play the chosen MP3 using a media player, how long will it take to play it all?

**Reply by Navid200** [reply]
This could also happen if you use the ascending volume profile.

**Comment by noctividus** [comment]
I have it set as the xDrip default sound. I would describe it as near
constant sound with maybe 700ms of silence in between the end and beginning
as the sound loops. That goes on indefinitely until I open the app and
snooze the alert.

**Reply by Navid200** [reply]
That could be a bug.  But, since no one else has reported that, it is possible that we are talking abut different things.  Please lets talk about this a little bit more.

Would you please show snapshots of the following pages?

Settings -> Alarms and Alerts -> Glucose Alerts Settings
Identify the alert that has this problem and show its specific settings page.
Please show the logs, only one image, containing the time the alarm triggers.

**Comment by noctividus** [comment]
![Screenshot_20211030-075506_xDrip+](https://user-images.githubusercontent.com/5158351/139539089-d2d21469-8d02-4816-9123-dc07867390ee.jpg)
![Screenshot_20211030-080209_xDrip+](https://user-images.githubusercontent.com/5158351/139539093-1a95c30a-7618-4c4d-8f93-34fe441593ba.jpg)
![Screenshot_20211030-080511_xDrip+](https://user-images.githubusercontent.com/5158351/139539109-85ceb1d2-ca43-4cfb-8d6f-7cbbf282e126.jpg)

**Reply by Navid200** [reply]
With those settings, after this particular alert triggers, it should wait about 4 minutes before going off if you don't snooze it.
But, if you have set 5 recently, please restart the phone.  There are some xDrip settings that will come from memory until a restart refreshes them.  This may be one of those settings.
After you restart the phone and the next time this alert triggers, please don't snooze right away.  Does it trigger right away instead of waiting 4 minutes?

**Comment by noctividus** [comment]
This alert has been unchanged for months- if not longer. It has remained
unchanged through several restarts.

Again, the sound plays once, then immediately plays again and again. It
loops continuously with only a slight pause (less than 1 second) before the
sound plays again.

For those 2 minutes in the log between the alert firing at 19:14 and the
snooze at 19:16, the speaker was continuously playing the sound until I got
to the phone.

**Reply by Navid200** [reply]
Are there other alerts?  Is it possible that there are multiple alerts, with the same sound chosen, triggering at the same time?
Can you look at Settings -> Alarms and Alerts -> Glucose Level Alerts List

**Reply by Navid200** [reply]
Would you please also show a snapshot of the classic system status page?

**Reply by noctividus** [reply]
Yes, there is another high alert that triggers at 180. This alert has a different sound, so it can't be confused.
Another thing to note, that when I test the 180 alert using the "Test" button, it alerts once. However, when I test the offending alert, it goes on forever until I hit snooze.

![Screenshot_20211030-085004_xDrip+](https://user-images.githubusercontent.com/5158351/139540346-67defa52-8d00-4aa1-a6ea-d0f575d3d4f0.jpg)

**Comment by noctividus** [comment]
![Screenshot_20211030-090239_xDrip+](https://user-images.githubusercontent.com/5158351/139540592-b0dccae4-093e-42ec-90f8-a5da28f3d420.jpg)

**Reply by Navid200** [reply]
I don't know why this is happening.
Please post here:
https://github.com/NightscoutFoundation/xDrip/issues/1418

**Reply by noctividus** [reply]
"no one else has reported that" Well, it looks like it was reported, but then went silent, and you closed it.

I would recommend not closing issues just because you want to keep your list clean or don't want to deal with the issue. That's what filters and labels are for. You closed two issues (now duplicates) for a very real bug. With a less tenacious person, that would have just slipped away and you would still have a bug.

Generally, when searching in GitHub, you don't include closed issues, because if an issue is closed, it means it's resolved. That's the generally accepted understanding closed = not an issue anymore. You're saying closed = I don't want to look at it. That's why it's the default search pattern to only search open issues. 

Not everyone has the time to hand-hold a gatekeeper through proving out a bug. Like, if someone is posting on GitHub, they are likely beyond the "did you try unplugging it?" phase of troubleshooting.

**Reply by Navid200** [reply]
If people don't have time, they can leave.  We haven't twisted anyone's arm to use xDrip.
I have no idea why anyone would think that because they don't have time, they can keep complaining and we will have time to clean up.  The developers are not paid to administer these issues.  We are all volunteers.

We are asking the user community to be part of the process by helping us sort out the issues.  We are asking them not to keep opening issues without first spending time figuring out if there is already an open or closed issue.  And if they don't have time, they can post here in the  discussions and we will spend the time to figure out what the next step should be.

Thanks for the feedback.
You can also reach out and volunteer to help.  Sometimes it's easier and more effective to "do" rather than to "direct".

**Reply by noctividus** [reply]
Reporting Bugs takes time, and sometimes that's all that a person can contribute. That contribution should be valued. It's a data point. I wouldn't destroy or discourage data points.

**Reply by Navid200** [reply]
I never discourage datapoints.  We are asking users to provide their input here in the discussions rather than using the issues platform.
Thanks for proving feedback and the time you spent on it.  There will be no destruction of any data.

**Comment by Navid200** [comment]
What happens if you edit your super High alert and temporarily, for test, switch the sound file to your Whistle sound file?
Will it still keep going when triggered?
